README.md

```diff
@@ -12,13 +12,32 @@ Please ensure you have the following installed:
 
 ### Checkout and View Local
 
+Clone this repo, and the Book theme as a submodule:
+
 ```bash
```

Comment:
The README mentioned not to use the `bash` code annotation and use `console` instead, but this was pre-existing. Wasn't sure if there was a good reason for it or if I should have updated it :)

Reply:
:heavy_check_mark: 392fbfb